### PR TITLE
Add initial feature set

### PR DESCRIPTION
**Pull request by mattburgess**

### CODE REVIEW

File: vpc_endpoints.tf [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+resource "aws_security_group" "vpc_endpoints" {
+  name        = "vpc-endpoints-${var.vpc_name}"
+  description = "Allows instances to reach Interface VPC endpoints"
+  vpc_id      = "${aws_vpc.vpc.id}"
+  tags        = "${var.common_tags}"
+}
+
+resource "aws_security_group_rule" "vpce_kafka_https_ingress" {
```
Comment (dan-hill2802) [inline vpc_endpoints.tf]: kafka shouldn't be in the resource name

File: vpc_endpoints.tf [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  description              = "Accept VPCE traffic"
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 443
+  to_port                  = 443
+  source_security_group_id = "${element(var.interface_vpce_source_security_group_ids, count.index)}"
+  security_group_id        = "${aws_security_group.vpc_endpoints.id}"
+}
+
+resource "aws_security_group_rule" "kafka_vpce_https_egress" {
```
Comment (dan-hill2802) [inline vpc_endpoints.tf]: kafka shouldn't be in the resource name

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  description = "(Required) The CIDR block of the VPC."
+}
+
+variable "vpc_instance_tenancy" {
+  default     = "default"
+  description = "(Optional) Tenancy of instances spun up within the VPC (`default`, `dedicated`. Defaults to `default`)"
+}
+
+variable "region" {
+  description = "(Optional) The region in which to deploy the VPC."
```
Comment (dan-hill2802) [inline variables.tf]: marked optional, but no default. Maybe grab the caller id's region?

**Review by dan-hill2802 — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 3 above)
Comment (mattburgess) [reply variables.tf]: Fixed by removing the optional label.

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**